dhis-2/dhis-services/dhis-service-validation/src/main/java/org/hisp/dhis/dataanalysis/jdbc/JdbcDataAnalysisStore.java

```diff
@@ -287,4 +302,90 @@ public JdbcDataAnalysisStore( StatementBuilder statementBuilder,
 
         return jdbcTemplate.query( sql, new DeflatedDataValueNameMinMaxRowMapper( null, null ) );
     }
+
+    /**
+     * HQL query used to select {@link FollowupValue}s.
+     */
```

Comment:
Wondering if it is better to put this in a dedicated, new class e.g. `FollowupValueManager` or similar. The idea is to get rid of everything else in `JdbcDataAnalysisStore` soon, and then the removal process is just about deleting that class entirely.

Reply:
done